### ISSUE TITLE
support `eslintrc.js` instead of deprecated `.eslintrc`

### ISSUE THREAD

**Issue by frederik-b** [AtomLinter/linter-eslint#1340]
## Issue Type
config options not supported or settings text incorrect

## Issue Description

This is the options text:

**Disabling**
[x] Disable when no ESLint config is found (in package.json or .eslintrc)

Now either it just dosn't work wir `.eslintrc.js` and `.eslintrc.json` or the description is wrong.
Either way `.eslintrc` is deprecated and this should be improved.

## Bug Checklist
- [ ] Check if `.eslintrc.js` is working and enabling the linter
- [ ] Check if `.eslintrc.json` is working and enabling the linter
- [ ] Check if Disabling option has correct text

```
Linter Eslint: Debug output here
```

**Comment by IanVS** [comment]
Good catch, the description is just wrong.  We'd happily take a PR to improve it.
